src/test/java/easytests/api/v1/controllers/SubjectsControllerTest.java

```diff
@@ -0,0 +1,53 @@
+package easytests.api.v1.controllers;
+
+/**
+ * @author VeronikaRevjakina
+ */
+public class SubjectsControllerTest {
+    /**
```

Comment:
paste comments same as in controller, do not show each method we should test.

Reply:
@malinink improved